src/Vegetation/BigLeaf/bigleaf.jl

```diff
@@ -0,0 +1,112 @@
+module BigLeaf
+using ClimaLSM
+using DocStringExtensions
+using ClimaCore
+import ...Parameters as LSMP # not sure if we need
+import ClimaCore: Fields, Operators, Geometry, Spaces
+# Not sure what we will need
+# NOTE: I feel like we won't be using any prognostic for now 
+# vegetation is static (LAI, roots, stem...) so we're not tracking a state
+# photosynthesis and transpiration are just computed from drivers (temperature etc)
+import ClimaLSM.Domains: AbstractDomain
+import ClimaLSM:
+    AbstractModel,
+    make_update_aux,
+    prognostic_vars,
+    auxiliary_vars,
+    name,
+    prognostic_types,
+    auxiliary_types
+export BigLeafModelParameters,
+    BigLeafModel
+
+"""
+"""
+struct BigLeafModelParameters{FT <: AbstractFloat, PSE} # Farquhar et al. (1980) 
+    # Radiative Transfer
+    "Leaf Area Index"
+    LAI::FT
+    "Vegetation exctinction coefficient"
+    K::FT
+    "PAR canopy reflectance"
+    ρ_leaf::FT
+    "Clumping index following Braghiere (2021)"
+    Ω::FT
+    # Photosynthesis
+    "Vcmax at 25 °C"
+    Vcmax25::FT
+    "leaf internal carbon dioxide partial pressure"
+    ci::FT
+    "CO2 compensation point"
+    cp::FT
+    "Moisture stress factor"
+    β::FT
+    "Michaelis-Menten parameter for CO2 at 25 °C"
+    Kc25::FT
+    "Michaelis-Menten parameter for O2 at 25 °C"
+    Ko25::FT
+    "Reference temperature"
+    To::FT
+    "Universal gas constant"
+    R::FT
+    "parameter for kc"
+    ΔHkc::FT
+    "parameter for ko"
+    ΔHko::FT
+    # Stomatal conductance
+    "Relative diffusivity of water vapor"
+    Drel::FT
+    "Slope parameter, inversely proportional to the square root of marginal water use efficiency"
+    g1::FT
+end
+
+# Environmental drivers: Shortwave radiation (SW) or PAR (~SW/2), air temperature (Ta), vapor pressure deficit (VPD)
+
+"""
+"""
+function BigLeafModelParameters{FT}(;
+
+) where {FT, PSE}
+return BigLeafModelParameters{FT, PSE}(
+
+)
+end
+
+"""
+"""
+abstract type AbstractVegetationBigLeafModel{FT} <: ClimaLSM.AbstractModel{FT} end
+
+"""
+"""
+struct BigLeafModel{FT, PS, D, DT} <:
```

Comment:
I am wondering if instead we want a 
`CanopyModel` with fields for radiative_transfer, photosynthesis, stomatal conductance, and plant_hydraulics. And then we choose a model for each of those - and right now we are choosing BeerLambert, Farquar, Medlyn, e.g?

Reply:
I think that is a good idea too.

Reply:
I renamed BigLeaf to Canopy, including for the module name, we could also change the folder name and think of structure of src/Vegetation at some point [0a9d5c4](https://github.com/CliMA/ClimaLSM.jl/commit/0a9d5c480ac81c0ccb96bfe6691b6142ac8079ec)